sdk/core/core-http/lib/webResource.ts

```diff
@@ -28,8 +28,6 @@ export type TransferProgressEvent = {
  */
 export interface AbortSignalLike {
   readonly aborted: boolean;
-  dispatchEvent: (event: Event) => boolean;
-  onabort: ((this: AbortSignalLike, ev: Event) => any) | null;
```

Comment:
Why are we still defining AbortSignalLike from `core-http` instead of using it from abort-controller?

Reply:
I left it mainly to avoid potentially sidespread changes for the interface import but it seems quite minimal.  I've updated the PR to re-export `AbortSignalLike` from `@azure/abort-controller`.